packages/backend/src/app.ts

```diff
@@ -477,6 +477,7 @@ export class App {
     // Read incoming payment
     router.get<DefaultState, SubresourceContextWithAuthenticatedStatus>(
       '/incoming-payments/:id',
+      createWalletAddressMiddleware(),
```

Comment:
If this middleware is already fetching the resources, does it make sense to also then call `incomingPaymentRoutes.get`? 

(applies to all of them)

Reply:
We _could_ (this isn't great per se) store the fetched resource right on the context, and the skip the second lookup if we already have something defined in the ctx

Reply:
Please create issue for that @mkurapov